.github/workflows/format.yml

```diff
@@ -0,0 +1,45 @@
+name: Format
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  edit:
+    name: clang-format
+    if: ${{ github.event.comment.body == '@votca-bot format' }}
+    runs-on: ubuntu-latest
+    container: votca/buildenv:latest
```

Comment:
I don't like the fact that pulling the container takes 60sec out of the 62sec runtime. 
Maybe we should make a small container, which has nothing but clang-format in there.

Reply:
If it's not a hassle I'm in agreement.

Reply:
It is easy.

Reply:
I agree

Reply:
https://github.com/votca/buildenv/pull/86, please review